### ISSUE TITLE
vitpose and pretrained backbone

### ISSUE THREAD

**Issue by xungeer29** [geopavlakos/hamer#34]
Thanks for your great work.
I want to train hamer with small parameter sizes, such as change hamer's backbone to vit-small. But I don't have the pretrained backbone, small hamer's performance is very poor.
I noticed that the pretrained backbone and vitpose-huge-wholebody you provided is slightly different from the official vitpose repo https://github.com/ViTAE-Transformer/ViTPose.
Can you provide the conversion method for the pretrained backbone network?
In other words, how can we obtain pretrained backbone of vitpose-small, vitpose-base and vitpose-large?

**Comment by geopavlakos** [comment]
The pretrained backbone is the same as the one provided by ViTPose (it should be available through [this link](https://1drv.ms/u/s!AimBgYV7JjTlgS5rLeRAJiWobCdh?e=41GsDd)). The only change we did was removing the `backbone.` prefix for each dictionary key, but the weights should be identical. You can similarly try with different backbones (although the code currently only defines the network architecture for ViT-H, so you probably need to do more edits there too).

**Comment by xungeer29** [comment]
> The pretrained backbone is the same as the one provided by ViTPose (it should be available through [this link](https://1drv.ms/u/s!AimBgYV7JjTlgS5rLeRAJiWobCdh?e=41GsDd)). The only change we did was removing the `backbone.` prefix for each dictionary key, but the weights should be identical. You can similarly try with different backbones (although the code currently only defines the network architecture for ViT-H, so you probably need to do more edits there too).

Thanks for your reply, I got it.

**Comment by geopavlakos** [comment]
We follow [instructions by the authors](https://github.com/ViTAE-Transformer/ViTPose/issues/64#issuecomment-1428036106) to split the model they provide into different versions (i.e., for wholebody pose, for animal pose, etc) and keep the wholebody component. If you follow the model splitting procedure, you should get an identical model with the one we provide.

Please note that we only use the wholebody model for the demo code, to get an estimate of the hand location in the image. We do not train/finetune that version.

**Comment by xungeer29** [comment]
> We follow [instructions by the authors](https://github.com/ViTAE-Transformer/ViTPose/issues/64#issuecomment-1428036106) to split the model they provide into different versions (i.e., for wholebody pose, for animal pose, etc) and keep the wholebody component. If you follow the model splitting procedure, you should get an identical model with the one we provide.
> 
> Please note that we only use the wholebody model for the demo code, to get an estimate of the hand location in the image. We do not train/finetune that version.

Thank you for your prompt reply and it solves my problem. I wish you make greater progress in your future research.
